src/parser.rs

```diff
@@ -212,27 +212,36 @@ impl<'a> Parser<'a> {
 
     pub fn path(self) -> Result<'a, String> {
         let (path, parser) = self.word()?;
-        let mut path = Path::new(path);
 
-        // For Linux and BSD and MacOS.
+        let path = Path::new(path);
+        if path.to_str().is_none() {
+            return Ok((String::from(""), parser));
+        }
+
+        let mut path_string = String::from(path.to_str().unwrap());
```

Comment:
There's no need to unwrap `path.to_str()` here if you match on both cases on line 217.

Reply:
Done.